ecosystem/platform/server/app/controllers/projects_controller.rb

```diff
@@ -4,20 +4,31 @@
 # SPDX-License-Identifier: Apache-2.0
 
 class ProjectsController < ApplicationController
-  before_action :authenticate_user!
   before_action :ensure_projects_enabled!
-  before_action :ensure_confirmed!
+  before_action :authenticate_user!, except: %i[index show]
+  before_action :ensure_confirmed!, except: %i[index show]
   respond_to :html
 
   # GET /projects
   def index
-    @projects = Project.where(public: true).all
+    @categories = Category.all.index_by(&:id)
+    @projects = Project.where(public: true).includes(:project_categories)
```

Comment:
should we have kaminari (or whatever pagination soup du jour is) on here?

Reply:
The way I imagined it was the index page would not have pagination, it would just show the top/latest projects, but you could still search for specific projects. Maybe we can add pagination - TBD.